docs/src/architecture/backend_architecture.md

```diff
@@ -26,31 +26,25 @@ The architecture of the server should be as simple as possible.
 We aim for the API provided to clients during a game to be as simple as possible:
 
 - move(joystick_values): Allows a player to move through the board by providing joystick values corresponding to the direction of movement.
-- basic_attack(joystick_values): Takes joystick values to determine the direction of the attack.
-- skill(skill_name, joystick_values): Accepts joystick values for skill direction along with the skill name.
+- use_skill(skill_name, joystick_values): Accepts joystick values for skill direction along with the skill name.
 - refresh: Used to request the entire game state.
 
 Additionally, the server can communicate various messages to the clients:
 
-- game_start: Indicates the start of the game.
-- game_end: Signals the end of the game.
-- state: Represents the differences between two states.
-- game_state: Contains all the information about the game.
-- ping: Used to inquire about the server's ping.
+- game_started: Indicates the start of the game.
+- game_finished: Signals the end of the game.
+- state_update: Represents the differences between two states.
+- player_joined: Indicates a player joining the game.
+- ping_update: Used to inquire about the server's ping.
 
 ## Matchmaking
 
-For matchmaking to work, players should be able to:
+For matchmaking to work, players should be able to join a game, which will automatically start once the lobby is full or we reach a time limit (whichever happens first). 
 
-- Join an existing lobby.
-- Create a lobby if no other exists.
-- Start the game once the lobby is full.
+There is a single process taking care of the matchmaking, the `MatchingCoordinator`. This process is responsible for keeping track of the lobby and its state, launching a new game when the lobby is full or the time limit is reached. It also provides an API for joining and leaving the lobby.
```

Comment:
While we still have a LobbyWebsocket, the lobbies themselves don't exist anymore. The MatchingCoordinator works as a sort of unique queue that receives all players that press the play button and once it reaches the max capacity for a game (or a time limit), it launches a game with those players (and possibly bots)